docs/ja/API-reference.md

```diff
@@ -482,49 +480,52 @@ dayjs().isAfter(dayjs()); // false
 dayjs().isAfter(dayjs(), 'year'); // false
 ```
 
-### Is a Dayjs `.isDayjs(compared: any)`
+#### Is a Dayjs
```

Comment:
We should update the table of contents accordingly, to ensure the anchor works.

Reply:
Oops, fixed.